src/Definition.php

```diff
@@ -89,4 +89,11 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Parameters array (for methods and functions), for use in textDocument/signatureHelp
+     *
+     * @var string[]
```

Comment:
If I understand your code correctly this string array only contains the pretty-printed parameter node. That means you cannot construct the `documentation` property from it, which you need to read from the `@param` tag. Of course as a new feature it will use more memory.